app/controllers/game_search_controller.rb

```diff
@@ -15,13 +15,37 @@ def index
   end
 
   def create
+    if !params[:search_type] && !params[:similar]
+      flash[:error] = "Please Select a Search Type"
+      redirect_to "/"
+    elsif params[:search] == ""
+      flash[:error] = "Please Enter a Valid Search"
+      redirect_to "/"
+    else
+      game = create_game_objects(params)
+      create_redirect(game)
+    end
+  end
```

Comment:
I like this work around instead of creating an entire new page for this one thing